src/picosvg/svg.py

```diff
@@ -201,10 +200,12 @@ def _is_removable_group(el):
     # no attributes makes a group meaningless
     if len(el.attrib) == 0:
         return True
-    return len(el) <= 1 or _opacity(el) in {0.0, 1.0}
+    num_children = sum(1 for e in el if e.tag is not etree.Comment)
 
+    return num_children <= 1 or _opacity(el) in {0.0, 1.0}
 
-def _try_remove_group(group_el):
+
+def _try_remove_group(group_el, push_attrib=False):
```

Comment:
looks like `push_attrib` is never used with False? Also, it's not clear from the name what the parameter does, it's actually only "pushing the opacity" from group down to its children, not a generic "attrib". I suggest removing it